src/PortToDocs/src/libraries/Configuration.cs

```diff
@@ -587,22 +586,28 @@ public static Configuration GetCLIArguments(string[] args)
                 Log.ErrorAndExit("You missed an argument value.");
             }
 
-            if (config.DirsDocsXml == null)
+            if (config.IncludedAssemblies.Count == 0)
             {
-                Log.ErrorAndExit($"You must specify a path to the dotnet-api-docs xml folder using '-{nameof(Mode.Docs)}'.");
+                Log.ErrorAndExit($"You must specify at least one assembly with {nameof(IncludedAssemblies)}.");
             }
 
-            if (config.DirsIntelliSense.Count == 0)
+            return config;
+        }
+
+        internal void VerifyIntellisenseXmlFiles()
+        {
+            if (DirsIntelliSense.Count == 0)
             {
                 Log.ErrorAndExit($"You must specify at least one IntelliSense & DLL folder using '-{nameof(Mode.IntelliSense)}'.");
             }
```

Comment:
These two checks were moved to their own methods. They are only invoked when the tool is executed relying on the filesystem.